### PR TITLE
Add Coffee Biscuits and Data and resolve contributors profiles

### PR DESCRIPTION
**Pull request by sitjart**

### CODE REVIEW

**Comment by sitjart** [comment]
I'll merge the branch since the issue comes from the resource and tool table.
I'll open an issue to remove that tool and resource table for now.

File: pages/training_materials/course_materials/coffee-biscuits-data.md [hunk 1]
```diff
@@ -10,9 +10,10 @@ toc: false
    <a class="home" href="https://docs.google.com/document/d/1ZRoCiX0fm7lUyCSsth8Zu46EOOoG2EO9NYPWDAkbD-4/edit?usp=sharing" target="_blank" rel="noopener noreferrer">
         <button type="button" class="home-button">Got to materials</button>
    </a>
+</html>
 <br>
 <br>
-</html>
+
 Coffee, Biscuits and Data is an awareness workshop, for both on-site and online, to build awareness of data management in research-performing organisations. The session is designed not to educate but to demystify data management while aiming to develop accountability – nearly the entire life sciences workforce is responsible for managing data in research.
```
Comment (poterlowicz-lab) [inline pages/training_materials/course_materials/coffee-biscuits-data.md:17]: I would remove "not to educate"

Comment (poterlowicz-lab) [reply pages/training_materials/course_materials/coffee-biscuits-data.md:17]: so it should be "The session is designed to demystify data management....."

Comment (poterlowicz-lab) [reply pages/training_materials/course_materials/coffee-biscuits-data.md:17]: otherwise great

Comment (sitjart) [reply pages/training_materials/course_materials/coffee-biscuits-data.md:17]: I merged it before I saw your comment. I've removed that bit

**Comment by poterlowicz-lab** [comment]
[like]  Krzysztof Poterlowicz reacted to your message:
________________________________

**Pull request merged**